src/InboundServer/middlewares.js

```diff
@@ -331,11 +331,12 @@ const createJwsValidator = (logger, keys, exclusions) => {
  * @return {Function}
  */
 const applyState = (sharedState) => async (ctx, next) => {
+    const outboundRequests = (sharedState.conf && sharedState.conf.mutualTLS && sharedState.conf.mutualTLS.outboundRequests) || {};
     Object.assign(ctx.state, {
         ...sharedState,
         conf: {
             ...sharedState.conf,
-            tls: sharedState?.conf?.mutualTLS?.outboundRequests,
+            tls: outboundRequests,
```

Comment:
why this change?

Reply:
@bushjames The previous construct was throwing error as it appears not be valid JavaScript.